jobs/JEVS_WAFS_ATMOS_PLOTS

```diff
@@ -60,7 +68,7 @@ export COMINstat=$COMIN/stats/$COMPONENT  # will be appended with wafs.$VDATE or
 
 # COMOUT folder
 # evs_ver defined in versions/run.ver
-export COMOUT=${COMOUT:-$COMIN/$STEP/$COMPONENT/$RUN.$VDATE}
+export COMOUT=${COMOUT:-$(compath.py ${envir}/com/${NET}/${evs_ver})/$STEP/$COMPONENT/$RUN.$VDATE}
```

Comment:
@YaliMao-NOAA , this is close! Please change to:
`export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})/$STEP/$COMPONENT/$RUN.$VDATE}`

Reply:
@ShelleyMelchior-NOAA 
Should jobs/JEVS_WAFS_ATMOS_STATS have this change?
export COMOUT=${COMOUT:-$(compath.py -o ${NET}/${evs_ver})}